specutils/tests/test_loaders.py

```diff
@@ -870,6 +870,17 @@ def test_tabular_fits_compressed(compress, tmp_path):
     assert quantity_allclose(spec.flux, spectrum.flux)
 
 
+def test_tabular_fits_jwst():
+    """Test reading of tabular FITS file of a JWST spectrum."""
+    spec = Spectrum.read("https://bdnyc.s3.us-east-1.amazonaws.com/Beiler24/SDSSpJ1346-00NIR.fits")
```

Comment:
this spectrum has an uncertainty column; want to check that that gets loaded too?